lib/cheerio.js

```diff
@@ -13,7 +13,8 @@ var api = [
   require('./api/attributes'),
   require('./api/traversing'),
   require('./api/manipulation'),
-  require('./api/css')
+  require('./api/css'),
+  require('./api/serialize')
```

Comment:
Sure thing. This was originally named `api.serialize.js` by @jlep so I tried to keep it as consistent as possible.